test_app_root/app/BUILD

```diff
@@ -0,0 +1,9 @@
+node_library(name='app_root', dependencies=['test_app_root/app/src:src'])
+
+node_package(
+    name="app_root_bundle",
+    dependencies=[":app_root"],
+    artifact_paths=["public/bundle"],
+    app_root="test_app_root/app",
+    output_path="output",
+)
```

Comment:
Packaging this target results in the following traceback:

```
$ ./pants --print-stacktrace package test_app_root/app:app_root_bundle                                                                                                                                                                                    16:28:07.09 [INFO] Initialization options changed: reinitializing scheduler...
16:28:07.63 [INFO] Scheduler initialized.
16:28:07.68 [ERROR] 1 Exception encountered:

Engine traceback:
  in select
  in pants.core.goals.package.package_asset
  in sendwave.pants_node.package.node_project_package (test_app_root/app:app_root_bundle)
  in sendwave.pants_node.package.get_node_package_digest (test_app_root/app:app_root_bundle)
  in sendwave.pants_node.package.get_npm_package_files
Traceback (no traceback):
  <pants native internals>
Exception: String("Cannot strip prefix app from root directory (Digest with hash Fingerprint<5086ccf5f6558eb4e5316fd2af47320bc00dc9eb45da67f65fd807e840e349a6>) - root directory didn't contain a directory named app but did contain directory named: test_app_root")

```

Not quite sure what this means or how to debug it. It seems like `RemovePrefix` is somehow getting `app` instead of `test_app_root/app` as the prefix to strip; I wonder if this is related to `test_app_root` being included in `root_patterns`?

https://github.com/waveremit/pants-node/blob/6b995fd374dd66d08a04f01a3943b629007dd60e/pants_plugins/sendwave/pants_node/package.py#L49